pkgs/by-name/li/libeufin/package.nix

```diff
@@ -16,12 +16,12 @@ let
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "libeufin";
-  version = "0.13.0";
+  version = "0.14.8";
 
   src = fetchgit {
```

Comment:
In `src.postFetch`, the `--short` flag should be removed, to ensure the output is reproducible.

Reply:
Or even better, turns out we can remove the whole thing and the version will still be correct even with a fake `HEAD.txt` file.

Reply:
> Or even better, turns out we can remove the whole thing and the version will still be correct even with a fake `HEAD.txt` file.

Turns out this doesn't really work on a newer libeufin release, so I've just removed the `--short` flag as you've suggested.